core/eolearn/core/constants.py

```diff
@@ -20,18 +20,24 @@
 TIMESTAMP_COLUMN = "TIMESTAMP"
 T = TypeVar("T")
 
+FEATURETYPE_DEPRECATION_MSG = (
+    "The `FeatureType.{}` has been deprecated and will be removed in the future. Use the EOPatch attribute `{}`"
+    " directly."
+)
+
 
 def _warn_and_adjust(name: T) -> T:
     # since we stick with `UPPER` for attributes and `lower` for values, we include both to reuse function
-    if name in ("TIMESTAMP", "timestamp"):
-        name = "TIMESTAMPS" if name == "TIMESTAMP" else "timestamps"
-
-    # if name.upper() in ("TIMESTAMPS", "BBOX"):
-    #     deprecation_msg = (
-    #         f"The `FeatureType.{name.upper()}` has been deprecated and will be removed in the future. Use the EOPatch"
-    #         f" attribute `{name.lower()}` directly."
-    #     )
-    #     warnings.warn(deprecation_msg, category=EODeprecationWarning, stacklevel=3)
+    if isinstance(name, str):  # to avoid type issues
+        if name in ("TIMESTAMP", "timestamp"):
+            name = "TIMESTAMPS" if name == "TIMESTAMP" else "timestamps"  # type: ignore[assignment]
+
+        if name in ("TIMESTAMPS", "BBOX", "timestamps", "bbox"):
```

Comment:
it's actually not the same. Here you assume that `name` is a string, but that is not the case. This is Enums we are talking about :D Hence the tests that all the things work as intended (because the code itself makes no such assurances)